src/data/queries/questionAnswer.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  QueryData,
+  QueryFormatter,
+  QueryOpts,
+  QueryParams,
+} from 'next-drupal-query'
+import { drupalClient } from '@/lib/utils/drupalClient'
+import { queries } from '.'
+import { NodeQA } from '@/types/dataTypes/drupal/node'
+import { QuestionAnswerType } from '@/types/index'
+import { generalEntityDataService } from '../delegators/generalEntityDataService'
+
+// Define the query params for fetching node--news_story.
+export const params: QueryParams<null> = () => {
+  return queries.getParams().addInclude([
+    'field_answer',
+    'field_buttons',
+    'field_related_benefit_hubs',
+    'field_related_information',
+    'field_tags.field_topics',
+    'field_tags.field_audience_beneficiares',
+    'field_tags.field_non_beneficiares',
+    // 'field_administration',
```

Comment:
These will be added when we have components and formatters for outputting these fields.